pkg/virtual/workspaces/registry/rest.go

```diff
@@ -154,6 +157,21 @@ func (s *REST) NamespaceScoped() bool {
 	return false
 }
 
+func (s *REST) isUserAllowed(user user.Info, name string, reviewer workspaceauth.Reviewer) (bool, error) {
+	review, err := reviewer.Review(name)
+	if err != nil {
+		return false, err
+	}
+	if review.EvaluationError() != nil {
+		return false, review.EvaluationError()
+	}
+	if !sets.NewString(user.GetGroups()...).HasAny(review.Groups()...) && !sets.NewString(review.Users()...).Has(user.GetName()) {
```

Comment:
have a hard time to understand what this means. Can you add a go doc oneliner explaining?

Reply:
This effectively is implementing a SAR with the data returned from the local reviewer.